MatrixKit/Controllers/MXKPreviewViewController.h

```diff
@@ -0,0 +1,74 @@
+/*
+ Copyright 2020 Vector Creations Ltd
```

Comment:
```suggestion
 Copyright 2020 The Matrix.org Foundation C.I.C
```

We have 2 kinds of copyright. Everything under https://github.com/matrix-org is transferred to the foundation.